### DISCUSSION TITLE
Mapping GO terms to descriptions and vice versa

### DISCUSSION THREAD

**Discussion by Ruiqi-CUB** [geneontology/helpdesk#430] [Q&A]
Hi all!

I have a list of GO terms and I would like to find the description of them. Is there a software or package that I can use? Also, how can I find the GO terms based on its description?

There are over 1000 GO terms in my list. I don't think I should do it manually on http://geneontology.org/

Thanks a lot!!

My GO list looks like this: [one GO term per line]

GO:0001505
GO:0006259
GO:0006403 ...

Ideal Output: (tab delimited)
GO:0001505                   regulation of neurotransmitter levels
GO:0001824                                  blastocyst development
GO:0002181                                 cytoplasmic translation

Thanks a lot

Ruiqi

**Comment by suzialeksander** [comment]
For this, any up-to-date InterMine will work- here's an example with YeastMine, which prominently displays the GO version. For a list of your size, upload the list (don't enter/paste them in).

> You can use the YeastMine Analyze tool available at SGD to retrieve the GO term names for each ID.
> 
> - Go to the [Analyze tool on YeastMine](http://yeastmine.yeastgenome.org/yeastmine/bag.do)
> - In the Select Type pull down, select GO Term
> - Enter your GO ids or upload a list in the full format (GO:0016020, GO:0016301…)
> - Click on Create List. The tool offers several options to download the list.
> 


http://geneontology.org/docs/faq/#how-do-i-get-the-term-names-for-my-list-of-go-ids

**Reply by Ruiqi-CUB** [reply]
Thank you so much! How about vice versa?

**Comment by Ruiqi-CUB** [comment]
Thank you so much! How about vice versa?

**Comment by suzialeksander** [comment]
https://yeastmine.yeastgenome.org/yeastmine/template.do?name=GOname_GOidentifier&scope=all
